Multisig/Cross-layer/Configuration/Config.Example.xcconfig

```diff
@@ -183,14 +183,7 @@ OTHER_LDFLAGS = $(inherited) -fprofile-instr-generate -ObjC
 WEB3AUTH_GOOGLE_CLIENT_ID = YOUR_WEB3AUTH_GOOGLE_CLIENT_ID
 // Set in https://dashboard.web3auth.io/home/customauth
 WEB3AUTH_GOOGLE_VERIFIER = YOUR_WEB3AUTH_GOOGLE_VERIFIER
-// Needs to be registered in entitlements of app
-WEB3AUTH_REDIRECT_URL = https:/$()/safe-wallet-web.staging.5afe.dev/web3auth/
-
-// Needs to be registered in entitlements of the app
-_WEB3AUTH_REDIRECT_URL_DEV = https:/$()/safe-wallet-web.dev.5afe.dev/web3auth/
-_WEB3AUTH_REDIRECT_URL_STAGING = https:/$()/safe-wallet-web.staging.5afe.dev/web3auth/
-_WEB3AUTH_REDIRECT_URL_PROD = https:/$()/app.safe.global/web3auth/
-WEB3AUTH_REDIRECT_URL = $(_WEB3AUTH_REDIRECT_URL_$(SERVICE_ENV):default=$(_WEB3AUTH_REDIRECT_URL_PROD))
+WEB3AUTH_REDIRECT_URL = YOUR_WEB3AUTH_REDIRECT_URL
```

Comment:
where is this value coming from?

Reply:
is jenkins configured to inject it?

Reply:
From Jenkins environment. Or originally the Google Oauth config. It is the value for "iOS URL scheme"